stl/inc/compare

```diff
@@ -98,32 +91,26 @@ public:
         // The stored value is either less (0xff), equivalent (0x00), greater (0x01), or unordered (0x80).
         // Subtracting from 0 produces either 0x01, 0x00, 0xff, or 0x80. Note that the effect is to
         // exchange less for greater (and vice versa), while leaving equivalent and unordered unchanged.
-        return partial_ordering{static_cast<_Compare_ord>(0 - static_cast<unsigned int>(_Val._Value))};
+        return partial_ordering{static_cast<_Compare_t>(0 - static_cast<unsigned int>(_Val._Value))};

```

Comment:
Can/should we say `return {MEOW};` here? Ditto below. I'm fine with `return TYPE{MEOW};` though.